tests/Propel/Tests/Runtime/Connection/ConnectionManagerPrimaryReplicaTest.php

```diff
@@ -12,22 +12,22 @@
 
 use Propel\Tests\Helpers\BaseTestCase;
 
-use Propel\Runtime\Connection\ConnectionManagerMasterSlave;
+use Propel\Runtime\Connection\ConnectionManagerPrimaryReplica;
 use Propel\Runtime\Adapter\Pdo\SqliteAdapter;
 
 use \PDO;
 
-class ConnectionManagerMasterSlaveTest extends BaseTestCase
```

Comment:
Please created another test for the ConnectionManagerPrimaryReplica.
Until ConnectionManagerMasterSlave is only deprecated this test should remain.